frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -379,6 +388,20 @@
           </AdminContent>
         </DetailItem>
       {/if}
+      <!-- Only show model version to project managers and admins, too technical to show to regular members -->
+      {#if project.type === ProjectType.FlEx && canManage}
+        <DetailItem title={$t('project_page.model_version')} text={flexModelVersion?.toString() ?? NULL_LABEL}>
```

Comment:
Not using `$number` here because that would have put commas in, and this number appears as 7000072 everywhere else it's used, not 7,000,072.